src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -749,6 +754,12 @@
         - name: --apiserver-subnet-id
           type: string
           short-summary: The ID of a subnet in an existing VNet into which to assign control plane apiserver pods(requires --enable-apiserver-vnet-integration)
+        - name: --enable-custom-ca-trust
+          type: bool
+          short-summary: Enable Custom CA Trust on agent node pool.
+        - name: --disable-custom-ca-trust --dcat
```

Comment:
About the failure in [linter](https://dev.azure.com/azure-sdk/public/_build/results?buildId=1589387&view=logs&j=ccdac6d3-465a-5ada-8819-d5b4a6b3a631&t=ffdbba7d-e523-5908-c7bf-0703bafb81b0&l=454), after checking the code, there is an implicit alphabetical sorting restriction. So, need to change this to `--dcat --disable-custom-ca-trust`.